examples/kafkatokafka/parser.go

```diff
@@ -0,0 +1,47 @@
+package kafkatokafka
+
+import (
+	"errors"
+
+	"github.com/digitalocean/captainslog"
+	"github.com/digitalocean/firebolt"
+	"github.com/digitalocean/firebolt/fbcontext"
+	log "github.com/sirupsen/logrus"
+)
+
```

Comment:
this is a very personal preference, but I'd love to see a 
```
var _ node.Node = Parser{}
```
to make it explicit which interface it's implementing

Reply:
fixed in next commit